functions-action.sln

```diff
@@ -0,0 +1,43 @@
+﻿
```

Comment:
I'm surprised to see a solution file getting added as part of this. Is this intentional?

Reply:
nope, not intentional...I guess it got added automatically - or I accidentally included it. Should I just remove it?